src/videojs.ga.coffee

```diff
@@ -66,11 +68,13 @@ videojs.plugin 'ga', (options = {}) ->
         seeking = true
         sendbeacon( 'seek start', false, seekStart )
         sendbeacon( 'seek end', false, seekEnd )
+        percentsAlreadyTracked = [];
 
     return
 
   end = ->
     sendbeacon( 'end', true )
+    percentsAlreadyTracked = [];
```

Comment:
well in fact when you finish to play the video and you replay it it's make sense to alos have analyse on the playback